File: fix-image-urls.php
<?php
$root = new RecursiveDirectoryIterator(__DIR__ . '/resources/views');
$iter = new RecursiveIteratorIterator($root);
$patterns = [
    '/src="\/images\/([^"]+)"/' => 'src="{{ asset(\'images/\1\') }}"',
    '/src="\.\/images\/([^"]+)"/' => 'src="{{ asset(\'images/\1\') }}"',
    '/src=\'\/images\/([^\']+)\'/' => 'src=\'{{ asset(\'images/\1\') }}\'',
    '/src=\'\.\/images\/([^\']+)\'/' => 'src=\'{{ asset(\'images/\1\') }}\'',
    '/image:\s*\'\/images\/([^\']+)\'/' => 'image: \'{{ asset("images/\1") }}\'',
    '/image:\s*\'\.\/images\/([^\']+)\'/' => 'image: \'{{ asset("images/\1") }}\'',
    '/\'image\'\s*=>\s*\'\/images\/([^\']+)\'/' => '\'image\' => asset(\'images/\1\')',
    '/\'image\'\s*=>\s*\'\.\/images\/([^\']+)\'/' => '\'image\' => asset(\'images/\1\')',
    '/\'logo\'\s*=>\s*\'\/images\/([^\']+)\'/' => '\'logo\' => asset(\'images/\1\')',
    '/\'logo\'\s*=>\s*\'\.\/images\/([^\']+)\'/' => '\'logo\' => asset(\'images/\1\')',
    '/\'countryFlag\'\s*=>\s*\'\/images\/([^\']+)\'/' => '\'countryFlag\' => asset(\'images/\1\')',
    '/\'countryFlag\'\s*=>\s*\'\.\/images\/([^\']+)\'/' => '\'countryFlag\' => asset(\'images/\1\')',
    '/\'userImage\'\s*=>\s*\'\/images\/([^\']+)\'/' => '\'userImage\' => asset(\'images/\1\')',
    '/\'userImage\'\s*=>\s*\'\.\/images\/([^\']+)\'/' => '\'userImage\' => asset(\'images/\1\')',
    '/\'flag\'\s*=>\s*\'\/images\/([^\']+)\'/' => '\'flag\' => asset(\'images/\1\')',
    '/\'flag\'\s*=>\s*\'\.\/images\/([^\']+)\'/' => '\'flag\' => asset(\'images/\1\')',
    '/img src="\.\/images\/([^"]+)"/' => 'img src="{{ asset(\'images/\1\') }}"',
    '/img src=\'\.\/images\/([^\']+)\'/' => 'img src=\'{{ asset(\'images/\1\') }}\'',
];
$changed = [];
foreach ($iter as $file) {
    if (!$file->isFile()) continue;
    if ($file->getExtension() !== 'php') continue;
    $path = $file->getPathname();
    if (strpos($path, DIRECTORY_SEPARATOR . 'views' . DIRECTORY_SEPARATOR) === false) continue;
    $content = file_get_contents($path);
    $original = $content;
    foreach ($patterns as $pattern => $replacement) {
        $content = preg_replace($pattern, $replacement, $content);
    }
    if ($content !== $original) {
        file_put_contents($path, $content);
        $changed[] = $path;
    }
}
echo 'changed ' . count($changed) . " files\n";
foreach ($changed as $path) {
    echo $path . "\n";
}

File: fix-image-urls.py
import re
from pathlib import Path
root = Path('resources/views')
replacements = [
    (re.compile(r'src="/images/([^"]+)"'), r'src="{{ asset(\'images/\1\') }}"'),
    (re.compile(r'src="\./images/([^"]+)"'), r'src="{{ asset(\'images/\1\') }}"'),
    (re.compile(r"src='/images/([^']+)'"), r"src='{{ asset(\'images/\1\') }}'"),
    (re.compile(r"src='\./images/([^']+)'"), r"src='{{ asset(\'images/\1\') }}'"),
    (re.compile(r"image:\s*'/images/([^']+)'"), r"image: '{{ asset(\"images/\1\") }}'"),
    (re.compile(r"image:\s*'\./images/([^']+)'"), r"image: '{{ asset(\"images/\1\") }}'"),
    (re.compile(r"'image'\s*=>\s*'/images/([^']+)'"), r"'image' => asset('images/\1')"),
    (re.compile(r"'image'\s*=>\s*'\./images/([^']+)'"), r"'image' => asset('images/\1')"),
    (re.compile(r"'logo'\s*=>\s*'/images/([^']+)'"), r"'logo' => asset('images/\1')"),
    (re.compile(r"'logo'\s*=>\s*'\./images/([^']+)'"), r"'logo' => asset('images/\1')"),
    (re.compile(r"'countryFlag'\s*=>\s*'/images/([^']+)'"), r"'countryFlag' => asset('images/\1')"),
    (re.compile(r"'countryFlag'\s*=>\s*'\./images/([^']+)'"), r"'countryFlag' => asset('images/\1')"),
    (re.compile(r"'userImage'\s*=>\s*'/images/([^']+)'"), r"'userImage' => asset('images/\1')"),
    (re.compile(r"'userImage'\s*=>\s*'\./images/([^']+)'"), r"'userImage' => asset('images/\1')"),
    (re.compile(r"'flag'\s*=>\s*'/images/([^']+)'"), r"'flag' => asset('images/\1')"),
    (re.compile(r"'flag'\s*=>\s*'\./images/([^']+)'"), r"'flag' => asset('images/\1')"),
    (re.compile(r"img src=\"\./images/([^"]+)\""), r"img src=\"{{ asset('images/\1') }}\""),
    (re.compile(r"img src='\./images/([^']+)'"), r"img src='{{ asset('images/\1') }}'"),
]
changed = []
for path in root.rglob('*.blade.php'):
    text = path.read_text(encoding='utf-8')
    original = text
    for pat, rep in replacements:
        text = pat.sub(rep, text)
    if text != original:
        path.write_text(text, encoding='utf-8')
        changed.append(path)
print(f'changed {len(changed)} files')
for p in changed:
    print(p)
